gym_sapientino/core/states.py

```diff
@@ -104,8 +118,8 @@ def to_dict(self) -> Tuple[Dict, ...]:
         """Encode into a dictionary."""
         return tuple(
             {
-                "discrete_x": math.floor(r.x),
-                "discrete_y": math.floor(r.y),
+                "discrete_x": round(r.x),
+                "discrete_y": round(r.y),
```

Comment:
Actually, now that I'm thinking about, I'm considering if the previous one was correct. I need to re-execute the map and see. There would be differences only with continus actions however.

Reply:
Yes, now I remember: integer values on the grid correspond to the center of grid cells, not one of the borders.
That's why I propose this change.